src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -223,13 +215,6 @@ public String getRepositoryUri(@NonNull BitbucketRepositoryType type,
         }
     }
 
-    @CheckForNull
-    public String getLogin() {
```

Comment:
Removed public method.

Adding it back in likely won't have the exact same semantics since I think we'd only be able to return the credential ID from the authenticator instead of the actual username. Would that still be acceptable?

Reply:
Seems you provided a deprecated method, resolving conversation